hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoUpdateHandler.java

```diff
@@ -202,13 +203,19 @@ private Account.Builder updateBuilder(
             builder.declineReward(op.declineReward().booleanValue());
         }
         if (op.hasStakedAccountId()) {
+            // 0.0.0 is used a sentinel value for removing staked account id
```

Comment:
In both the sentinel `if` blocks we will instead reset this `oneof staked_id` to `UNSET` once https://github.com/hashgraph/pbj/issues/160 is closed, right?

Reply:
Yes agreed, will add comment.